test/module/ForwardFlow.Module.ts

```diff
@@ -73,22 +89,23 @@ describe("NEW::: Smart Account Forward Flow", async () => {
     const charlieBalanceBefore = await charlie.getBalance();
     const tokenAmountToTransfer = ethers.utils.parseEther("0.167924");
     
-    const { transaction, feeRefund, signature } = await buildecdsaModuleAuthorizedForwardTx(
+    const { transaction, feeRefund, signature } = await buildEcdsaModuleAuthorizedForwardTx(
       charlie.address,
       "0x",
       userSA,
       smartAccountOwner,
       ecdsaModule.address,
-      tokenAmountToTransfer.toString(),
+      forwardFlowModule,
+      tokenAmountToTransfer.toString()
     );
     
     await expect(
-      userSA.execTransaction_S6W(transaction, feeRefund, signature)
+      forwardFlowModule.execTransaction(userSA.address, transaction, feeRefund, signature)
```

Comment:
so since this is enabled module via module manager, it internally makes call smarTaccount.execTransactionFromModule

but module manager is also used to add validation plugins right, which plays a role in AA transaction. just trying to fully unerstand

Reply:
Yes, Module Manager can enables any module, be it the validation module (mostly used for AA flow, and just doing validation) or any other module, that can be called directly and can do execution through `execTransactionFromModule`